examples/integrations/coherence/helidon-creditscore-mp/src/test/java/io/helidon/bestbank/creditscore/MainTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.bestbank.creditscore;
+
+import jakarta.inject.Inject;
+import jakarta.ws.rs.client.WebTarget;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.client.Entity;
+import jakarta.ws.rs.core.MediaType;
+
+import io.helidon.microprofile.testing.junit5.HelidonTest;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@HelidonTest
+class MainTest {
+
+    @Inject
+    private WebTarget target;
+
+    @Test
+    void testCreditScore() {
+        try (Response r = target
+                .path("creditscore")
+                .request()
+                .post(Entity.entity("""
+                                {
+                                    "ssn" : "123-45-6789",
+                                    "firstName" : "Frank",
+                                    "lastName" : "Helidon",
+                                    "dateOfBirth" : "02/19/2019"
+                                }
+                                """, MediaType.APPLICATION_JSON)
+                )){
```

Comment:
Formatting is weird, you can use a variable for the JSON payload to make it prettier.

Reply:
Fixed